src/containerapp/azext_containerapp/_clients.py

```diff
@@ -1018,7 +1018,7 @@ def list(cls, cmd, resource_group_name, environment_name):
 
 
 class SessionPoolPreviewClient():
-    api_version = SESSION_API_VERSION
+    api_version = PREVIEW_API_VERSION
```

Comment:
Do you need to change this api-version? 
As you change this version, you will need to rerun other tests that relate to `az sessionpool` with --live.